schema/notification.js

```diff
@@ -0,0 +1,30 @@
+const BaseModel = require('./base-model');
+const { uuid } = require('../lib/regex-validation');
+
+class Notification extends BaseModel {
+  static get tableName() {
+    return 'notifications';
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      additionalProperties: false,
+      properties: {
+        id: { type: 'string', pattern: uuid.v4 },
+        to: { type: 'string' },
```

Comment:
Do we want to enforce email format here, as in https://github.com/UKHomeOffice/asl-schema/blob/master/schema/profile.js#L76